week-1/exercises/paths.js

```diff
@@ -14,5 +14,15 @@ const lastSlashIndex = filePath.lastIndexOf("/");
 const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
+// Extract the "dir" part
+const dir = filePath.slice(0, lastSlashIndex);
+
+// Extract the "ext" part
+const dotIndex = base.lastIndexOf(".");
+const ext = dotIndex !== -1 ? base.slice(dotIndex + 1) : "";
```

Comment:
again, this work, but there are simpler ways of performing the same functionality. I would argue that `base.split(".").slice(-1)[0]` is easier to read and understand.